File: Aula_3/Exercicio8/app/src/main/java/br/edu/decision/exercicio8/MainActivity.java
package br.edu.decision.exercicio8;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import java.util.ArrayList;
import java.util.Arrays;

public class MainActivity extends AppCompatActivity {
    private ArrayList<String> planetList;
    private ArrayAdapter<String> adapter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ListView listView = (ListView) findViewById(R.id.listView);

        String[] planetas = new String[]{"Mercúrio", "Vênus", "Terra", "Marte", "Júpiter", "Saturno", "Urano", "Netuno"};
        planetList = new ArrayList<String>();
        planetList.addAll(Arrays.asList(planetas));

        adapter = new ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, android.R.id.text1, planetList);
        listView.setAdapter(adapter);
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu){
        getMenuInflater().inflate(R.menu.menu, menu);
        return super.onCreateOptionsMenu(menu);
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item){

        if(item.getItemId() == R.id.btn_adicionar){
            planetList.add("Plutão");
        }
        if(item.getItemId() == R.id.btn_remover){
            planetList.remove(0);
        }
        adapter.notifyDataSetChanged();

        return super.onOptionsItemSelected(item);
    }
}

File: Aula_3/Exercicio7/app/src/main/java/infnet/edu/br/exercicio7/MainActivity.kt
package infnet.edu.br.exercicio7

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.widget.ArrayAdapter

import kotlinx.android.synthetic.main.activity_main.*
import java.util.ArrayList

class MainActivity : AppCompatActivity() {

    var planetList =  ArrayList<String>()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val array = listOf("Mercúrio","Vênus","Terra","Marte","Júpiter","Saturno","Urano","Netuno")

        planetList.addAll(array.toList())

        listView.adapter = ArrayAdapter<String>(this,android.R.layout.simple_list_item_1,android.R.id.text1,planetList)

    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {

        menuInflater.inflate(R.menu.manu2,menu)
        return super.onCreateOptionsMenu(menu)

    }

    override fun onOptionsItemSelected(item: MenuItem?): Boolean {

        when(item?.itemId){
            R.id.mn_add -> addPlaneta()
            R.id.mn_delete -> deletPlaneja()

        }

        return super.onOptionsItemSelected(item)
    }

    private fun addPlaneta() {
        planetList.add("Novo1")
        (listView.adapter as ArrayAdapter<String>).notifyDataSetChanged()
    }

    private fun deletPlaneja() {
        planetList.removeAt(0)
        (listView.adapter as ArrayAdapter<String>).notifyDataSetChanged()
    }




}
